### PR TITLE
Fixed issue 13971 Country values missing in admin customer grid for multi-store configuration

### PR DESCRIPTION
**Pull request by Dharmeshvaja91**
### Fixed Issues (if relevant)

1. magento/magento2#13971: Country values missing in admin customer grid for multi-store configuration
### Manual testing scenarios (*)

1. Manual Testing

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @Dharmeshvaja91. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/Directory/Model/ResourceModel/Country/Collection.php [hunk 1]
```diff
@@ -187,6 +187,19 @@ public function loadByStore($store = null)
         return $this;
     }
 
+    /**
+     * Load allowed countries for all stores
+     *
+     * @param null|int|string|\Magento\Store\Model\Store $store
+     * @return \Magento\Directory\Model\ResourceModel\Country\Collection
+     */
+    public function loadByAllStore($store = null)
```
Comment (slavvka) [inline app/code/Magento/Directory/Model/ResourceModel/Country/Collection.php:196]: 1. This is [backward incompatible change](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/) as this class has `@api` annotation and you introduced a public method.
1. Does that mean that for any customer there will be shown all countries that even maybe absent for a specific store?

**Review by slavvka — CHANGES_REQUESTED** [review]
Hey @Dharmeshvaja91 Thank you for the contribution. Please look at the comments.

**Comment by sidolov** [comment]
@Dharmeshvaja91 , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Comment by sidolov** [comment]
@Dharmeshvaja91 will you continue progress on this PR?

**Comment by Dharmeshvaja91** [comment]
> @Dharmeshvaja91 will you continue progress on this PR?

yes i am working on this.

**Comment by sidolov** [comment]
@Dharmeshvaja91 , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**